moz_security/tests/integration/analysis_webinj/run/input/generate_webinj.cfg

```diff
@@ -0,0 +1,22 @@
+filename = "file.lua"
+input_filename = "run/input/access.log"
+
+decoder_module = "decoders.moz_logging.line_splitter"
```

Comment:
You don't need to use the line splitter (that is only required for the stream kinesis)